### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by muelletm** [OpenBMB/BMTools#41]
Hi!

Thanks for creating and maintaining this effort!

In the paper (https://arxiv.org/pdf/2304.08354.pdf) it says *The codes and datasets are publicly
available for further research exploration*, but I didn't find any mention to the data in the repo.
Has the dataset been released?

Thanks a lot!

**Comment by thuqinyj16** [comment]
Thanks for your interests in our work, we have recently uploaded part of the data to [this link] (https://cloud.tsinghua.edu.cn/d/2dab79f7b66841329f45/), and will continually upload more. 

In addition, we have created training data for some tools at [this link] (https://cloud.tsinghua.edu.cn/d/7447641dd09e4b8280bd/), they can also be used for testing. Welcome if you also want to collaborate on building an open-source tool learning-relevant SFT data.

If there's any follow-up question, feel free to ask.
